.github/workflows/test-performance-command.yml

```diff
@@ -1,4 +1,4 @@
-name: performance-test
+name: Run Performance Test
```

Comment:
```suggestion
# runs ./tools/bin/ci_performance_test.sh
# which is more or less ./gradlew performanceTest limited to connectors with changes
name: Run Performance Test
```

might be worth adding to the text of test too.  I'm not sure where it would be most useful to our users